app/javascript/packages/webauthn/webauthn-input-element.ts

```diff
@@ -13,12 +13,14 @@ export class WebauthnInputElement extends HTMLElement {
     return this.hasAttribute('show-unsupported-passkey');
   }
 
-  toggleVisibleIfPasskeySupported() {
+  async toggleVisibleIfPasskeySupported() {
     if (!this.hasAttribute('hidden')) {
       return;
     }
-
-    if (isWebauthnPasskeySupported()) {
+    const isPublicKeyCredentialSupported =
+      window.PublicKeyCredential &&
+      (await window.PublicKeyCredential.isUserVerifyingPlatformAuthenticatorAvailable());
```

Comment:
Not strictly necessary, but since we already have a pattern for extracting logic of `isWebauthnPasskeySupported()` to its own file with its own specs, I wonder if we should do the same with `isPublicKeyCredentialSupported` for consistency?